shape.lic

```diff
@@ -152,6 +152,7 @@ class Shape
       stow_item(@noun)
     when /trash/
       dispose_trash(@noun)
+      stop_script('workorders')
```

Comment:
Same with `exit`. See https://github.com/rpherbig/dr-scripts/issues/952

Reply:
It is unintuitive to me that the 'lower' script (`shape`) would stop its 'calling' script (`workorders`). I think a Flag in `workorders` would be a better choice, so it can watch for low quality items.

Reply:
Working on a better method.